source/core/Events.cpp

```diff
@@ -540,7 +550,7 @@ struct EventSpec {  // Specifier for Event structure configuration data
 typedef EventSpec *EventSpecRef;
 
 // GetEventSourceForEventType -- return the EventSource associated with an event type
-static inline EventSource GetEventSourceForEventType(EventType eType) {
+static EventSource GetEventSourceForEventType(EventType eType) {
```

Comment:
Leave inline.

Reply:
Even though the inline is unnecessary because it is a global function?

Reply:
It's static because it doesn't need to be extern, it's not used outside this file, and it's inline because I want to instruct the compiler to inline it if possible.
Many compilers will already try to inline static functions if possible, but it's still useful to provide the hint.
We could use an anonymous namespace instead of static for local scope, but we haven't been using that technique elsewhere.